### ISSUE TITLE
jupyter book dependencies

### ISSUE THREAD

**Issue by ianm99** [UBC-MDS/GoT-fatality-prediction#42]
I'm running this so that I can build the jupyter book:

conda install -c conda-forge jupyter-book

Does the container need to be updated accordingly?

**Comment by 786213750** [comment]
yes, I will update the container all together. Can you send me version of jupyter-book that you r using?
